exporter/awscloudwatchlogsexporter/config.go

```diff
@@ -42,6 +44,21 @@ type Config struct {
 	// e.g. logs.us-east-1.amazonaws.com
 	// Optional.
 	Endpoint string `mapstructure:"endpoint"`
+
+	// QueueSettings is a subset of exporterhelper.QueueSettings,
+	// because only QueueSize is user-settable due to how AWS CloudWatch API works
+	QueueSettings struct {
+		// QueueSize set the lenght of the sending queue
+		QueueSize int `mapstructure:"queue_size"`
+	} `mapstructure:"sending_queue"`
+}
+
+// Validate config
+func (config *Config) Validate() error {
+	if config.QueueSettings.QueueSize < 1 {
+		return errors.New("sending_queue.queue_size must be 1 or greater")
+	}
```

Comment:
should force `NumConsumers` always to be 1 in Validate() to prevent unexpected misconfiguration. There is no way to send logs to CW in parallel for a single log stream due to the sequence token.

Reply:
User won't be able to set `NumConsumers` as it's not exposed by `QueueSettings` type defined here. To that seems like a good way to avoid confusion - what do you think?